nested/db-mysql.json

```diff
@@ -47,7 +47,7 @@
             },
             "resources": [
                 {
-                    "apiVersion": "2017-12-01",
+                    "apiVersion": "latest",
```

Comment:
Let's not use "latest" for apiVersion, let's have a unique date string around this commit/time-frame for every applicable mention of apiVersion.